contracts/eden/src/actions/elect.cpp

```diff
@@ -74,6 +75,20 @@ namespace eden
       elections.seed(btc_header);
    }
 
+   void eden::electmeeting(eosio::name account,
+                           uint8_t round,
+                           const std::vector<encrypted_key>& keys,
+                           const eosio::bytes& data)
+   {
+      eosio::require_auth(account);
+      members members{get_self()};
+      elections elections{get_self()};
+      auto group_id = elections.get_group_id(account, round);
+      members.check_keys(elections.get_group_members(group_id), keys);
```

Comment:
Since a group member without a key stops the group from using this feature, should we require members set keys to be eligible to vote?

Reply:
I don't think that use of this feature should be mandatory.  It's only a convenience.  If it's enforced at all, it should be enforced by the front end, not the contract.